lib/phoenix_live_view.ex

```diff
@@ -1569,7 +1569,7 @@ defmodule Phoenix.LiveView do
     opts = Keyword.merge(opts, id: Phoenix.LiveView.Utils.random_id())
 
     socket
-    |> Phoenix.Component.assign_new(:streams, fn -> %{__changed__: MapSet.new()} end)
+    |> Phoenix.Component.do_assign_new(:streams, fn -> %{__changed__: MapSet.new()} end)
```

Comment:
We should avoid injecting `@doc false` functions in `Phoenix.Component` because they can still be imported into user code! What do you think about:

1. Make Phoenix.Component.assign always validate
2. Have a non-validating version in `Phoenix.LiveView.Utils.assign`
3. Change this code to call `Phoenix.LiveView.Utils.assign` instead

Reply:
Done! 👌